tests/scripts/tmate-pod.yaml

```diff
@@ -0,0 +1,79 @@
+# Apply this manifest to CI clusters to allow debugging complex failures during CI runs
```

Comment:
To be clear, these steps need to be added to the github action? Then how does the dev know how to connect?

Reply:
Yes. The connection info is printed by getting the `kubectl logs` of the pod